slave/components/quality_controller/error_correction.go

```diff
@@ -21,16 +32,100 @@ type errorCorrection struct {
 }
 
 // NewErrorCorrection constructs and returns an errorCorrection struct, which implements the Controller interface
-func NewErrorCorrection(ctx context.Context, dockerCli *client.Client, config *config_parser.Config, fileToDecontaminate string) Controller {
+func NewErrorCorrection(ctx context.Context, dockerCli *client.Client, config *config_parser.Config, fileToCorrect string) Controller {
 	return &errorCorrection{
 		ctx:             ctx,
 		dockerCLI:       dockerCli,
 		config:          config,
-		toDecontaminate: fileToDecontaminate,
+		toDecontaminate: fileToCorrect,
+	}
+}
+
+// TODO: Replace this function into a constants file where this can be used by other quality control processes
+// getImageID attempts to find the Docker image by given term
+func getImageID(client *client.Client, ctx context.Context, term string) (string, error) {
```

Comment:
`getImageID` is already available in this package; there is no need to re-define it here. See my comment on #70